guides/source/active_record_querying.md

```diff
@@ -1486,19 +1486,19 @@ books.each do |book|
 end
 ```
 
-The above code will execute just **2** queries, as opposed to **11** queries in the previous case:
+The above code will execute just **2** queries, as opposed to the **11** queries from the original case:
```

Comment:
We should also change line 1412 in the `includes` section to match this one.

Reply:
Ah, thanks for catching this one!

Reply:
Fixed.